labcodes/confetti-storybook
# [CSYS-360] Migrating Radio component

Link to task:
[CSYS-360](https://labcodes.atlassian.net/browse/CSYS-360)

Staging app: https://csys-359-migrate-icon--confetti-storybook.netlify.app/

How to test:
After running your project locally, run the Radio.test.js to check if the Alert component is working as expected.

Description of your solution:

Removing previous mentions from Class Components and moving the Props outside the functional component scope.

Screenshots (when applicable): NA
